src/pages/storage/StorageDetail.tsx

```diff
@@ -54,40 +68,28 @@ const StorageDetail: FC = () => {
         <div className="p-panel__content">
           <NotificationRow />
           <Row>
-            <table className="storage-detail-table">
-              <tbody>
-                <tr>
-                  <th className="u-text--muted">Name</th>
-                  <td>{storagePool.name}</td>
-                </tr>
-                <tr>
-                  <th className="u-text--muted">Status</th>
-                  <td>{storagePool.status}</td>
-                </tr>
-                <tr>
-                  <th className="u-text--muted">Size</th>
-                  <td>
-                    <StorageSize storage={storagePool} />
-                  </td>
-                </tr>
-                <tr>
-                  <th className="u-text--muted">Source</th>
-                  <td>{storagePool.config?.source ?? "-"}</td>
-                </tr>
-                <tr>
-                  <th className="u-text--muted">Description</th>
-                  <td>
-                    {storagePool.description ? storagePool.description : "-"}
-                  </td>
-                </tr>
-                <tr>
-                  <th className="u-text--muted">Driver</th>
-                  <td>{storagePool.driver}</td>
-                </tr>
-              </tbody>
-            </table>
-            <h2 className="p-heading--5">Used by</h2>
-            <StorageUsedBy storage={storagePool} project={project} />
+            <Tabs
```

Comment:
Please make sure that only the tabs content is scrollable, and everything above is fixed at the top all the time, just like in the instance and profile detail pages. This is particularly important for the notification, because it's not being scrolled into view (intentionally? Or is it unexpected?), leaving the user confused with no apparent feedback from the operation. See the following screen capture to understand the problem more concretely:

![Peek 2023-08-07 18-48](https://github.com/canonical/lxd-ui/assets/56583786/16e7299a-2937-4d58-8eb3-f733f79ce93b)